prefab_cloud_python/config_client.py

```diff
@@ -116,12 +116,10 @@ def streaming_loop(self):
         auth = "%s:%s" % ("authuser", self.options.api_key)
         token = base64.b64encode(auth.encode("utf-8")).decode("ascii")
         headers = {
-            "x-prefab-start-at-id": self.config_loader.highwater_mark,
+            "x-prefab-start-at-id": f"{self.config_loader.highwater_mark}",
             "Authorization": "Basic %s" % token,
         }
-        response = urllib3.PoolManager().request(
-            "GET", url, headers=headers, preload_content=False
-        )
+        response = requests.get(url, headers=headers)
```

Comment:
similar to client.py, lets instantiate a Session (either our own or reuse the one from Client)